pyproject.toml

```diff
@@ -47,12 +47,12 @@ attrs = ">=19.3.0"
 natsort = "5.5.0"
 arrow = "0.12.1"
 PyQtChart = "5.14.0"
-pywin32 = { version = "223", markers = "sys_platform == 'win32'" }
+pywin32 = { version = "225", markers = "sys_platform == 'win32'" }
```

Comment:
I assume pywin32 upgrade from 223 to 225 is just to get it to work with Python 3.8?

Reply:
Yes, `pywin32` first supports `python3.8` in version `225`